app/layout/layout.phtml

```diff
@@ -84,14 +84,16 @@
 <?php
 	$msg = '';
 	$status = 'closed';
+	$notificationID = '';
 	$notif = Minz_Request::getNotification();
 	if (!empty($notif)) {
 		$msg = $notif['content'];
 		$status = $notif['type'];
+		$notificationID = $notif['notificationID'];
 		invalidateHttpCache();
 	}
 ?>
-<div role="dialog" id="notification" class="notification <?= $status ?>" aria-describedby="dialogMsg">
+<div role="dialog" id="notification" class="notification <?= $status ?> <?= $notificationID ?>" aria-describedby="dialogMsg">
```

Comment:
Given the context, something like `notificationType` might be better than ID?

Reply:
`type` is already used for `good`/`bad` CSS class:

https://github.com/FreshRSS/FreshRSS/blob/4ab47defd55676d294046ad140d962d115dee0f7/app/layout/layout.phtml#L85-L94

I would say that it is an notification identifier (ID). Better wording is welcome :)

Reply:
Notification name?

Reply:
Just basically anything that doesn't look like `class="<? $id ?>"`. :-P

Reply:
Notification class.

Reply:
Thanks! I chose `notificationName`